Modules/OrgUnit/_PublicApi/README.md

```diff
@@ -0,0 +1,22 @@
+## Public Service
+Usage:
+```
+$orgunit = new OrgUnitUserService();
+$orgu_users = $orgunit->getUsers([292,291]);
+```
+This will return a list of ilOrgUnitUser. The Positions and the Superios were not loaded. You may load them by:
+```
+foreach($orgu_users as $orgu_user) {
+	/**
+	 * @var OrgUnit\User\ilOrgUnitUser $user
+	 */
+	print_r($orgu_user->getSuperiors());
```

Comment:
It would be great if you could (prosaically) describe how the superiors are determined. The business logic here could be interesting even for non developers to reach a mutual understanding of the word `Superior` .